packages/store/tests/state.spec.ts

```diff
@@ -84,4 +85,107 @@ describe('Store', () => {
       expect(res).toBe(true);
     });
   });
+
+  describe('given the ngxsOnInit lifecycle method is present', () => {
+    it('should call the ngxsOnInit method on root module initialisation', () => {
```

Comment:
can remove this test. there is already a test for ngxsOnInit in state.spec

Reply:
Yes, I saw that when I re-based. I would like to keep this test because it corresponds to other parts of the test describe section to show completeness. I will look at your test and see which to include.

Reply:
Ok. If this covers the same same remove the other so we don't just have a dupe test